Tests/SwerverTests/ResponseSpec/HttpResponseSpec.swift

```diff
@@ -5,48 +5,113 @@ import Foundation
 
 class HttpResponseSpec: QuickSpec {
     override func spec() {
-        describe("HttpResponse") {
-            var responseBuilder = ResponseBuilder()
+        describe("A Reponse object") {
+            func buildBuilder() -> HttpResponseBuilder {
+                return HttpResponseBuilder()
+            }
             
-            it("can equate two response object") {
-                let responseOne = responseBuilder
+            func defaultResponseBuilder() -> HttpResponse {
+                return HttpResponseBuilder()
                     .with(statusCode: .ok)
                     .with(contentType: .text)
-                    .with(body: String())
+                    .set(header: .allow, value: "GET")
+                    .with(body: "Hello")
                     .build()
```

Comment:
This line should be removed. That way, existing tests can continue setting attributes.

Reply:
fixed